src/site/content/en/blog/coop-coep/index.md

```diff
@@ -47,16 +48,51 @@ mitigate that risk, browsers offer an opt-in-based isolated environment called
 cross-origin isolated. With a cross-origin isolated state, the webpage will be
 able to use privileged features including:
 
-* [`SharedArrayBuffer`](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/SharedArrayBuffer)
-  (required for WebAssembly Threads. This is available from Android Chrome 88.
-  Desktop version is currently enabled by default with the help of [Site
-  Isolation](https://www.chromium.org/Home/chromium-security/site-isolation),
-  but will require the cross-origin isolated state and will be disabled by
-  default.)
-* [`performance.measureUserAgentSpecificMemory()`](/monitor-total-page-memory-usage/) (Ends its
-  origin trial and is planned to be enabled by default in Chrome 88)
-* [JS Self-Profiling API](https://wicg.github.io/js-self-profiling/) (Not
-  available yet in Chrome)
+<div class="w-table-wrapper">
+  <table>
+    <thead>
+      <tr>
+        <th>API</th>
```

Comment:
We might add performance.now() and performance.timeOrigin here as well?

https://bugs.chromium.org/p/chromium/issues/detail?id=1180178

Reply:
I did once, but removed because I wasn't too sure about these. I'll do.

Reply:
Fixed. Please have a look.
https://deploy-preview-4813--web-dev-staging.netlify.app/coop-coep/